src/Microsoft.ML.Recommender/RecommenderCatalog.cs

```diff
@@ -87,6 +87,7 @@ internal RecommendationTrainers(RecommendationCatalog catalog)
             /// <format type="text/markdown">
             /// <![CDATA[
             ///  [!code-csharp[MatrixFactorization](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Recommendation/MatrixFactorizationWithOptions.cs)]
+            ///  [!code-csharp[MatrixFactorization](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Recommendation/OneClassMatrixFactorizationWithOptions.cs)]
```

Comment:
You may want to check this won't generate too much content for the user. I had 4 links for time series but after speaking with Natalie I reduced to one.

Reply:
Matrix factorization works extremely differently with different loss functions. We must have two samples.

---
In reply to: [274225365](https://github.com/dotnet/machinelearning/pull/3282#discussion_r274225365) [](ancestors = 274225365)